### PR TITLE
Move node/master sizes (from kubemark jobs) to config file (so that they are next to --gcp-nodes)

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

**Comment by shyamjvs** [comment]
Need to fix presubmits.

**Comment by shyamjvs** [comment]
Let's hold this until we get https://github.com/kubernetes/test-infra/pull/7710 rolled out.

/hold

**Comment by shyamjvs** [comment]
You can now use flags instead of env vars, now that my PR #7710 is rolled
out.

**Comment by wojtek-t** [comment]
> You can now use flags instead of env vars, now that my PR #7710 is rolled out.

It's not yet rolled out: https://github.com/kubernetes/test-infra/pull/7716

**Comment by shyamjvs** [comment]
Sorry - I think I was looking at the wrong bump PR which is earlier one
(7689).

**Comment by wojtek-t** [comment]
Anyway - I've updated the PR. Once the changes are rolled out, this would be ready.
@shyamjvs - PTAL

File: jobs/env/pull-kubernetes-kubemark-e2e-gce.env [hunk 1]
```diff
@@ -1,2 +0,0 @@
-### cluster-env
```
Comment (shyamjvs) [inline jobs/env/pull-kubernetes-kubemark-e2e-gce.env]: Have you deleted this file or just the lines?

Comment (wojtek-t) [reply jobs/env/pull-kubernetes-kubemark-e2e-gce.env]: whole file

**Pull request merged**